tests/rgb/integration/drain.rs

```diff
@@ -0,0 +1,67 @@
+#![cfg(not(target_arch = "wasm32"))]
+use anyhow::Result;
+use bitmask_core::{
+    bitcoin::{drain_wallet, get_wallet_data, save_mnemonic},
+    structs::SecretString,
+};
+
+use crate::rgb::integration::utils::{send_some_coins, ISSUER_MNEMONIC, OWNER_MNEMONIC};
+
+#[tokio::test]
+#[ignore = ""]
+pub async fn drain() -> Result<()> {
+    // 1. Initial Setup
+    let old_keys = save_mnemonic(
```

Comment:
Always get a "empty wallet", because drain wallet get all utxos related with descriptor.

```suggestion
     let old_keys = new_mnemonic(&SecretString("".to_string())).await?;
```